sabnzbd/newswrapper.py

```diff
@@ -139,6 +139,10 @@ def con(sock, host, port, sslenabled, write_fds, nntp):
                 except WantReadError:
                     select.select([sock], [], [], 1.0)
 
+            # Log SSL/TLS info
+            logging.info("%s@%s: Connected using %s (%s)",
```

Comment:
It's only done when the connection is opened, which is only once per NZB, so very little.
I also noticed that for example with my 50 connections to Frugal, not all used the exact same cipher.